src/network/netdev/tunnel.c

```diff
@@ -617,6 +691,38 @@ int config_parse_encap_limit(const char* unit,
         return 0;
 }
 
+int config_parse_erpsan_index(const char* unit,
+                             const char *filename,
+                             unsigned line,
+                             const char *section,
+                             unsigned section_line,
+                             const char *lvalue,
+                             int ltype,
+                             const char *rvalue,
+                             void *data,
+                             void *userdata) {
+        Tunnel *t = userdata;
+        uint32_t k = 0;
+        int r;
+
+        assert(filename);
+        assert(lvalue);
+        assert(rvalue);
+
+        r = safe_atou32(rvalue, &k);
+        if (r < 0) {
+                log_syntax(unit, LOG_ERR, filename, line, r, "Failed to parse erspan index option, ignoring: %s", rvalue);
+                return 0;
+        }
+
+        if (k >= (1 << 20) || k == 0)
+                log_syntax(unit, LOG_ERR, filename, line, 0, "Invalid erspan index %d. Ignoring", k);
+        else
```

Comment:
Can you comment on why `k==0` is not allowed? https://patchwork.ozlabs.org/patch/800327/ seems to allow that.

Reply:
I do not follow the patchwork threads, but the final version (v3?) does not accept index 0.
https://git.kernel.org/pub/scm/network/iproute2/iproute2.git/commit/ip?id=9a1381d50913d59aa3ba1ccb12dc3a0bbd64573b